code/modules/mob/living/simple_animal/abnormality/_auxiliary_modes/joke/aleph/carrey.dm

```diff
@@ -0,0 +1,75 @@
+/mob/living/simple_animal/hostile/abnormality/carrey
+	name = "A Song For A Season"
+	desc = "<span class='bolddanger' style='font-size:16px;'>The ice is melting...</span>"
+	health = 1000 // Dummy
+	maxHealth = 1000 // Dummy
+	pixel_x = -16
+	pixel_y = -32
+	base_pixel_x = -16
+	base_pixel_y = -32
+	icon = 'ModularTegustation/Teguicons/64x128.dmi'
+	icon_state = "carrey"
+	icon_living = "carrey"
+	portrait = "carrey"
+	del_on_death = FALSE
+	can_breach = FALSE // Not... yet
+	start_qliphoth = 1337 // Dummy
+	threat_level = ALEPH_LEVEL
+	fear_level = ALEPH_LEVEL
+	damage_coeff = list(RED_DAMAGE = 0, WHITE_DAMAGE = 0, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 1,
+		ABNORMALITY_WORK_INSIGHT = 1,
+		ABNORMALITY_WORK_ATTACHMENT = 1,
+		ABNORMALITY_WORK_REPRESSION = 1,
+		"Request Concert" = 999,
+	)
+	work_damage_amount = 1
+	work_damage_type = WHITE_DAMAGE
+	abnormality_origin = ABNORMALITY_ORIGIN_JOKE
+
+/mob/living/simple_animal/hostile/abnormality/carrey/Initialize(mapload)
+	. = ..()
+	start_qliphoth = days_until_christmas()
+	if(start_qliphoth > 0)
+		icon_state = "contained_greyscale"
+		if(start_qliphoth < 24)
+			add_overlay(mutable_appearance('ModularTegustation/Teguicons/64x128.dmi', "containmentice[round((start_qliphoth / 4), 1)]"))
+
+/mob/living/simple_animal/hostile/abnormality/carrey/PostSpawn()
+	. = ..()
+	for(var/turf/open/O in range(1, src))
+		new /obj/effect/snow_storm(O)
+
+/mob/living/simple_animal/hostile/abnormality/carrey/AttemptWork(mob/living/carbon/human/user, work_type)
+	if(work_type == "Request Concert")
+		to_chat(user, span_bolddanger("Not yet... I will show you my... grand performance in [start_qliphoth] days."))
+		return null
+	. = ..()
+
+/mob/living/simple_animal/hostile/abnormality/carrey/FailureEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
+	to_chat(user, "<span class='bolddanger' style='font-size: 16px;'>You can't take it anymore!</span>")
```

Comment:
I quite dislike doing `style='font-size: 16px;'`, its not exactly prohibited but should we really do it instead of using `userdanger` or `narsiesmall`?
Your choice ultimatelly on this one, its stylistic
```suggestion
	to_chat(user, "<span class='bolddanger' style='font-size: 16px;'>You can't take it anymore!</span>")
```